### PR TITLE
upgrade goldpinger version and increase ping/check timeout

### PR DESCRIPTION
**Pull request by ZetaoZhuang**
**Reason for Change**:

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

**Review by timraymond — DISMISSED** [review]
Changes seem reasonable to me 👍

File: test/integration/manifests/goldpinger/daemonset.yaml [hunk 1]
```diff
@@ -43,12 +43,12 @@ spec:
...
             - name: PORT
               value: "8080"
-            - name: PING_TIMEOUT_MS
-              value: "5000"
-            - name: CHECK_TIMEOUT_MS
-              value: "5000"
-            - name: CHECK_ALL_TIMEOUT_MS
-              value: "10000"
+            - name: PING_TIMEOUT
+              value: "10000ms"
```
Comment (timraymond) [inline test/integration/manifests/goldpinger/daemonset.yaml:47]:
```suggestion
              value: "10s"
```

File: test/integration/manifests/goldpinger/daemonset.yaml [hunk 2]
```diff
@@ -43,12 +43,12 @@ spec:
...
+            - name: CHECK_TIMEOUT
+              value: "20000ms"
```
Comment (timraymond) [inline test/integration/manifests/goldpinger/daemonset.yaml:49]:
```suggestion
              value: "20s"
```

File: test/integration/manifests/goldpinger/daemonset.yaml [hunk 3]
```diff
@@ -43,12 +43,12 @@ spec:
...
+            - name: CHECK_ALL_TIMEOUT
+              value: "20000ms"
```
Comment (timraymond) [inline test/integration/manifests/goldpinger/daemonset.yaml:51]:
```suggestion
              value: "20s"
```

**Review by timraymond — CHANGES_REQUESTED** [review]
Small request: If these configs now allow units, it would be more obvious to readers to use seconds instead of making them do the math.

**Review by estebancams — APPROVED** [review]

**Review by timraymond — APPROVED** [review]

**Pull request merged**